circle.yml

```diff
@@ -37,9 +37,6 @@ test:
   override:
     - ./bin/circleci/test-addon.sh
     - ./bin/circleci/test-frontend.sh
-    - sh ./bin/circleci/test-firefox-release.sh
```

Comment:
I'd agree with you if build time was the only concern, but these are causing builds to intermittently fail, and idempotency is an important quality of a good test suite.